assignment2/hmm_evaluation.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from sklearn.metrics import confusion_matrix
+
+#from mfcc_extract import load_mfccs
+
+class RecognitionEvaluation:
+    def __init__(self, ground_truth, predicted):
+    
+        self.ground_truth = ground_truth.split() # load mfcc of original speech
+        self.predicted = predicted.split() # predicted mfcc
+        self.N = len(self.ground_truth)
+
+        # Error counts
+        self.substitution_errors = 0
+        self.deletion_errors = 0
+        self.insertion_errors = 0
```

Comment:
this part doesn't quite make sense.

Reply:
this is store different errors but i forgot initialise